roles/vdm/tasks/storage.yaml

```diff
@@ -18,8 +18,10 @@
     existing: "{{ vdm_overlays }}"
     add:
       - { transformers: "cas-add-nfs-mount.yaml", vdm: true }
-      - { transformers: "compute-server-add-nfs-mount.yaml", vdm: true }
-      - { transformers: "launcher-service-add-nfs.yaml", vdm: true }
+      - { transformers: "compute-server-add-nfs-mount.yaml", max: "2021.1.6", vdm: true }
+      - { transformers: "compute-server-add-nfs-mount.v2.yaml", min: "2021.2", vdm: true }
```

Comment:
Guessing here that 2021.1.6 is the last 2021.1.x release and that 2021.2 is the first 2021.2.x release? Also should the 2021.2 value have a patch level associated with it? Did not know if we have a 2021.2.0 release.

Reply:
That's right, 2021.1.6 is the last 2021.1.x release.
2021.2 is the first 2021.2 release, it is an lts. There is no 2021.2.0 stable release, the first 2021.1.x stable is 2021.2.1 which is later than lts 2021.2 using our version comparisons so it would be selected with min "2021.2"